lib/spack/spack/binary_distribution.py

```diff
@@ -111,13 +111,14 @@ def write_buildinfo_file(prefix, workdir, rel=False):
         dirs[:] = [d for d in dirs if d not in blacklist]
         for filename in files:
             path_name = os.path.join(root, filename)
-            filetype = relocate.get_filetype(path_name)
-            if relocate.needs_binary_relocation(filetype, os_id):
-                rel_path_name = os.path.relpath(path_name, prefix)
-                binary_to_relocate.append(rel_path_name)
-            elif relocate.needs_text_relocation(filetype):
-                rel_path_name = os.path.relpath(path_name, prefix)
-                text_to_relocate.append(rel_path_name)
+            if relocate.strings_contains_installroot(path_name):
```

Comment:
@gartung Is this check required now that `patchelf('--print-rpath'` is surrounded with a try/except? I'm asking because it may be confusing to a reader why this check exists. A comment would be helpful, but if it is now handled in `get_existing_elf_rpaths` then IMO this check isn't needed here.

Reply:
I want to add a check to cut down on the number of files potentially needing relocation.